cookbooks/aws-parallelcluster-config/templates/default/init/fetch_and_run.erb

```diff
@@ -84,6 +104,9 @@ function run_postupdate () {
 check_params
```

Comment:
custom_script is not define when executing check_params

Reply:
Good catch! I've moved the `custom_script` declaration to be before `check_params`.